Tests/Test_UARTDeadLock.c

```diff
@@ -16,6 +17,10 @@ CPU_STK UART_Deadlocks2_Stk[512];
 int counter1 = 0;
 int counter2 = 0;
 
+void foo(){
```

Comment:
This doesn't really follow code guidelines but idk if it matters since it's just a return and a test file.

Reply:
we should just delete it if it's not called anywhere

Reply:
it doesn't